### PR TITLE
Update tests to include up to 3.10

### PR DESCRIPTION
**Pull request by emirkmo**
Hey, I just tested your package from 3.7-3.10 and set it up to automatically do so using GitHub Actions (not that we were expecting any breaking). It would be great if you could update pypi package version to reflect that? See my changes in this PR. Feel free to just take what you think is necessary.. Commits have been squashed before merging as I had to experiment a bit to get tox running with github actions using tox-gh-actions

### CODE REVIEW

**Comment by emirkmo** [comment]
Partially closes #19

**Comment by ForeverWintr** [comment]
Many thanks for taking the time to do this! I am not familiar with github actions, so it's great to have someone set it up. I will take some time to understand what this does. 

Why do you say this partially closes #19? What remains to be done?

File: tox.ini [hunk 1]
```diff
@@ -4,12 +4,22 @@
...
+    3.8: py38
+    3.9: py39
+    3.10: py310
+
 
 [testenv]
 deps =
     coverage
     ansicolors
+    colors.py
```
Comment (ForeverWintr) [inline tox.ini:22]: Why do we need this package?

Comment (emirkmo) [reply tox.ini:22]: You use it in your tests. (Import colors). At least the tests wouldn’t pass without it

**Comment by emirkmo** [comment]
> Many thanks for taking the time to do this! I am not familiar with github actions, so it's great to have someone set it up. I will take some time to understand what this does.
> 
> Why do you say this partially closes #19? What remains to be done?

Partially because it won’t be closed until pypi is updated, that’s all.

I am similarly inexperienced with GitHub actions but thought it was the best way to set it up moving forward

File: tox.ini (hunk 1 above)
Comment (ForeverWintr) [reply tox.ini:22]: I see. Thank you for adding it!

**Comment by ForeverWintr** [comment]
It seems my CI set up is out of date. I will merge this, and make some changes on my end. Thanks again!

File: tox.ini (hunk 1 above)
Comment (ForeverWintr) [reply tox.ini:22]: Actually, `ansicolors` is what `import colors` is referring to.

**Pull request merged**